WORKSPACE

```diff
@@ -15,9 +15,9 @@ ENVOY_REPO = "envoy"
 
 # https://github.com/envoyproxy/envoy/tree/v1.23.6
```

Comment:
nit: -> `v1.23.7`

Reply:
oops, I missed this last time as well, let me update to v1.23.x :see_no_evil: